src/index.html

```diff
@@ -8,7 +8,224 @@
     <link rel="stylesheet" href="./css/index.scss" />
   </head>
   <body>
-    <div id="main"></div>
+    <div id="main">
```

Comment:
This is a rather completed pull request overall. 
I like the indentation and comments. It is also well structure with some clever inline styles.
Well done Franklin!

Reply:
Hi @Codyzhao , the indentation is auto-formatted by Prettier when the pre-commit hook is triggered. I did that configuration when initialized the codebase: https://github.com/nzittraining/cv-assignment/blob/master/package.json#L17-L20

Reply:
Cool, thanks for your effort Nick!
No wonder all the indentations are perfect.

Reply:
No worries. 

I agree on that it's a relevant high quality PR. Well done @FranklinGuan .